File: app/src/main/java/com/stevenfrew/beatprompter/midi/ClockMessage.java
package com.stevenfrew.beatprompter.midi;

class ClockMessage extends OutgoingMessage
{
    private final static byte[] MIDI_CLOCK_SIGNAL_BYTES=new byte[]{0x0f,(byte)0xf8,0,0};
    ClockMessage()
    {
        super(MIDI_CLOCK_SIGNAL_BYTES,true);
    }
}

File: app/src/main/java/com/stevenfrew/beatprompter/midi/ClockMessage.kt
package com.stevenfrew.beatprompter.midi

internal class ClockMessage : OutgoingMessage(MIDI_CLOCK_SIGNAL_BYTES, true) {
    companion object {
        private val MIDI_CLOCK_SIGNAL_BYTES = byteArrayOf(0x0f, 0xf8.toByte(), 0, 0)
    }
}